### ISSUE TITLE
Function inputs not indented 

### ISSUE THREAD

**Issue by Roald87** [Barteling/STweep#47]
Not sure if this is intended, if a setting is wrong or this is a bug.

```
// in
CalculateSomething := Parameters[1]
	* Parameters[2]
	* SomeFunction(Parameters[3], Parameters[4], Parameters[5], Parameters[6], Parameters[7], Parameters[8], Parameters[9])	* 2;

// out
CalculateSomething := Parameters[1]
	* Parameters[2]
	* SomeFunction(
	Parameters[3],
	Parameters[4],
	Parameters[5],
	Parameters[6],
	Parameters[7],
	Parameters[8],
	Parameters[9]
)
	* 2;

// expected
CalculateSomething := Parameters[1]
	* Parameters[2]
	* SomeFunction(
		Parameters[3],
		Parameters[4],
		Parameters[5],
		Parameters[6],
		Parameters[7],
		Parameters[8],
		Parameters[9]
	)
	* 2;
```
- "STweep for TwinCAT 3.4.0
- [settings.txt](https://github.com/Barteling/STweep/files/6607948/settings.txt)

**Comment by Barteling** [comment]
Looks like a bug to me, Parameters[3] to [9] should definitely be indented from 'SomeFunction' 
Thanks for reporting.

**Comment by Barteling** [comment]
I fixed this in STweep for Codesys 3.6.1-beta and STweep for TwinCAT 3.6.3-Beta

https://www.stweep.com/beta-releases/
